providers/shared/components/s3/id.ftl

```diff
@@ -220,6 +220,12 @@
                         "Default" : "Daily"
                     }
                 ]
+            },
+            {
+                "Names" : "ObjectOwnership",
```

Comment:
Do we need a default here?

Reply:
I was leaving it off because I didn't want to trigger adding config for existing deployments. In reality, the default behaviour is "writer" so if you think it is better, we could always add the object ownership config, in which case we'd add that as the default.